client/my-sites/upgrades/domain-management/dns/office-365.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
+import FormButton from 'components/forms/form-button';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormFooter from 'my-sites/upgrades/domain-management/components/form-footer';
+import FormInputValidation from 'components/forms/form-input-validation';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import isEmpty from 'lodash/isEmpty';
```

Comment:
Small nitpick, but `lodash/isEmpty` (and `i18n-calypso` above) are external dependencies and should be moved the the block above :)